app/controllers/todos_controller.rb

```diff
@@ -9,6 +9,13 @@ def create
     render json: todo
   end
 
+  def toggle_status
+    todo = Todo.find(params[:id])
+    todo.completed = !(todo.completed)
```

Comment:
unnecessary parenthesis  
```suggestion
    todo.completed = !todo.completed
```